v5/core/utils.go

```diff
@@ -182,6 +188,20 @@ func GetCurrentTime() int64 {
 	return time.Now().Unix()
 }
 
+// ParseDate parses and returns a strfmt Date pointer
+func ParseDate(dateString string) *strfmt.Date {
```

Comment:
It seems like we should have an error return on this method.

Reply:
Updated with an error return!